django/contrib/postgres/functions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Classes that represent PostgreSQL database functions.
+"""
+from django.db.models.expressions import Func, Value
+
+
+class Similarity(Func):
+    """
+    Trigram similarity function.
+    """
+    function = 'SIMILARITY'
+
+    def __init__(self, expression, string, **extra):
+        if not hasattr(string, 'resolve_expression'):
+            string = Value(string)
+        super(Similarity, self).__init__(expression, string, **extra)
```

Comment:
This needs to set its `output_field` to something different. At present the best way to guarantee that is to pass it up to the super of `__init__` if not supplied. The correct output field in this case is a `FloatField`.